solidity_bridge/script/Deploy.s.sol

```diff
@@ -0,0 +1,66 @@
+pragma solidity ^0.8.24;
+
+import {Script, console} from "forge-std/Script.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+
+import {Test, console} from "forge-std/Test.sol";
+import {Verifier} from "../src/Verifier.sol";
```

Comment:
can't see it in code base. Have you removed it?

Reply:
I see, added here [37991e7](https://github.com/gear-tech/gear-bridges/pull/46/commits/37991e76edd5f618e7e6da20556d3ff6e4598704)